internal/gcsx/bucket_manager.go

```diff
@@ -40,6 +42,8 @@ type BucketConfig struct {
 	StatCacheCapacity                  int
 	StatCacheTTL                       time.Duration
 	EnableMonitoring                   bool
+	EnableStorageClientLibrary         bool
+	debugGcs                           bool
```

Comment:
why is everything else public and debugGcs private?

Reply:
It was previously initialised flag, I didn't need to change it. should I update it?

Reply:
But added as part of GoClient library changes onnly right. I think you need to make it public

Reply:
fixed